### PR TITLE
Soussecret

### PR DESCRIPTION
**Pull request by JC1738**
Print a warning if ENV variables contain work secret or password

### CODE REVIEW

**Comment by sous-machine-account** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">PR is classed as Work in Progress</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: cli/actions/manifest_set.go [hunk 1]
```diff
@@ -36,6 +37,11 @@ func (ms *ManifestSet) Do() error {
 
 	messages.ReportLogFieldsMessage("Manifest in Execute", logging.ExtraDebug1Level, ms.LogSink, yml)
 
+	badEnv := getBadKeys(yml, []string{"password", "secret"})
```
Comment (lachlanh) [inline cli/actions/manifest_set.go]: would add "key", "auth" to the list

Comment (samsalisbury) [reply cli/actions/manifest_set.go]: ..and maybe "pass", "credentials", "connection", "token", ... ?

**Comment by JC1738** [comment]
Going to close PR now until we investigate some ideas how to best to handle secrets.

**Pull request closed without merging**